src/router/openai_proxy_test.py

```
@@ -1,4 +1,49 @@
+
+    monkeypatch.setattr(ChatRequest, "model_validate_json", lambda body: None)
+    upstream_response = {"id": "resp1", "object": "chat.completion", "choices": []}
+    # route call the url
+    route = respx.post(UPSTREAM_URL).mock(
+        return_value=httpx.Response(200, json=upstream_response)
+    )
+    # resp proxy via app
+    resp = client.post("/chat/completions", json=valid_payload())
+
+    if resp.status_code != 200:
+        raise AssertionError(f"expected status 200, got {resp.status_code}")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)